src/Scene/NodeProcess.js

```diff
@@ -191,6 +192,14 @@ NodeProcess.prototype.refineNodeLayers = function refineNodeLayers(node, camera,
     }
 };
 
+NodeProcess.prototype.getContentLayers = function getContentLayers(node, camera, params) {
+    if ((node.content.size < params.layersConfig.getGeometryLayers().length) && node.pendingLayers[2] === undefined) {
```

Comment:
geometryLayers aren't all feature layers, so `.size` might always be `<` to the geometry layers count.